vespalib/src/vespa/vespalib/util/time.h

```diff
@@ -92,5 +92,7 @@ class Timer
  * The default frequency (1000hz) for vespa timer, with environment override VESPA_TIMER_HZ capped to [1..1000]
  */
 uint32_t getVespaTimerHz();
+duration adjustTimeoutByDetectedHz(duration timeout);
+duration adjustTimeoutByHz(duration timeout, long hz);
```

Comment:
Should these be in explicit milliseconds rather than `duration`?

Reply:
Trying to avoid explicit time units and just stick to duration.